tests/server/routes/get-index.js

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!sinon',
+  'intern/dojo/node!../../../server/lib/routes/get-index',
+], function (registerSuite, assert, sinon, route) {
+  var request, response, write;
+
+  registerSuite({
+    name: 'routes/get-index',
+
+    'interface is correct': function () {
+      assert.isFunction(route);
+      var instance = route();
+      assert.isObject(instance);
+      assert.lengthOf(Object.keys(instance), 3);
+      assert.equal(instance.method, 'get');
+      assert.equal(instance.path, '/');
+      assert.isFunction(instance.process);
+    },
+
+    'route.process': {
+      setup: function () {
+        request = {
+          headers: { 'user-agent': 'foo' },
+          query: {
+            campaign: 'bar',
+            context: 'baz',
+            entrypoint: 'qux',
+            migration: 'blee',
+            service: 'wibble',
+            zignore: 'ignore me'
+          }
+        };
+        response = { render: sinon.spy() };
+        write = process.stderr.write;
+        process.stderr.write = sinon.spy();
+        route().process(request, response);
+      },
+
+      teardown: function () {
+        process.stderr.write = write;
+      },
+
+      'response.render was called correctly': function () {
+        assert.equal(response.render.callCount, 1);
+        assert.equal(response.render.thisValues[0], response);
+        var args = response.render.args[0];
+        assert.lengthOf(args, 2);
+        assert.equal(args[0], 'index');
+        assert.isObject(args[1]);
+        assert.lengthOf(Object.keys(args[1]), 2);
+        assert.match(args[1].flowId, /^[0-9a-f]{32}$/);
+        assert.isAbove(args[1].flowTime, 0);
+      },
+
+      'process.stderr.write was called correctly': function () {
+        assert.equal(process.stderr.write.callCount, 1);
+        assert.isTrue(process.stderr.write.calledBefore(response.render));
+        assert.equal(process.stderr.write.thisValues[0], process.stderr);
+        var args = process.stderr.write.args[0];
+        assert.lengthOf(args, 1);
+        assert.equal(args[0].substr(0, 14), 'activityEvent ');
+        assert.equal(args[0][args[0].length - 1], '\n');
+        var eventData = JSON.parse(args[0].substr(14));
```

Comment:
Can you add a space above this line to indicate the portion below is testing a different item?